src/ds.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from "react";
+import Firestore from "./services/firestore";
+import { makeStyles } from "@material-ui/core/styles";
+import Card from "@material-ui/core/Card";
+import CardActionArea from "@material-ui/core/CardActionArea";
+import CardActions from "@material-ui/core/CardActions";
+import CardContent from "@material-ui/core/CardContent";
+import CardMedia from "@material-ui/core/CardMedia";
+import Button from "@material-ui/core/Button";
+import Typography from "@material-ui/core/Typography";
+
+class Grid extends Component {
```

Comment:
Not quite sure what this file is doing... Regardless, I do not see the point of having hard coded values like "kricheso@google.com".